From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oleg <balbes-150@yandex.ru>
Date: Thu, 2 Feb 2023 14:08:11 +0100
Subject: [ARCHEOLOGY] fix Station P1 (#4796)

> X-Git-Archeology: - Revision 50a1ecd2f673f55017318c44d5fb1d1b8a51a5d9: https://github.com/armbian/build/commit/50a1ecd2f673f55017318c44d5fb1d1b8a51a5d9
> X-Git-Archeology:   Date: Thu, 02 Feb 2023 14:08:11 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: fix Station P1 (#4796)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3399-roc-pc-plus.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-roc-pc-plus.dts b/arch/arm64/boot/dts/rockchip/rk3399-roc-pc-plus.dts
index 7ba1c28f70a9..68594178a79a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-roc-pc-plus.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-roc-pc-plus.dts
@@ -133,6 +133,10 @@ &i2s0_8ch_bus {
 		<3 RK_PD7 1 &pcfg_pull_none>;
 };
 
+&i2s0 {
+	pinctrl-names = "default";
+};
+
 &i2s1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2s_8ch_mclk_pin>, <&i2s1_2ch_bus>;
-- 
Armbian

